automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -184,6 +191,7 @@ def satellite6_capsule_upgrade(admin_password=None):
     print('Wait till packages update ... ')
     print('YUM UPDATE started at: {0}'.format(time.ctime()))
     update_packages(quiet=False)
+    setup_capsule_firewall()
```

Comment:
Better to put this above register command itself before starting any upgrade task on capsule.

Still, if you are ok here, put this after the below print line which says the yum update finished at <time>.